conf/repos.yaml.template

```
@@ -2,7 +2,7 @@ REPOS:
   SATELLITE_VERSION_UNDR: "@jinja {{this.robottelo.satellite_version | replace('.', '_')}}"
   # RHEL major version can be used by some repositories to frame their repository URL for
   # extended RHEL versions.
-  RHEL_MAJOR_VERSION: "@jinja {{this.server.version.rhel_release[0]}}"
+  RHEL_MAJOR_VERSION: "@jinja {{this.server.version.rhel_version[0]}}"
```

we updated this in satelliteqe-jenkins already,
```suggestion
  RHEL_MAJOR_VERSION: "@jinja {{this.server.version.rhel_version | int}}"
```